taxi/forms.py

```diff
@@ -0,0 +1,31 @@
+from django.contrib.auth import get_user_model
+from django.contrib.auth.forms import UserCreationForm
+from django import forms
+
+from taxi.models import Driver, Car
+from taxi.validators import DriverValidationLicenseNumber
+
+
+class DriverCreationForm(UserCreationForm, DriverValidationLicenseNumber):
+    class Meta(UserCreationForm.Meta):
+        model = Driver
+        fields = (UserCreationForm.Meta.fields
+                  + ("first_name", "last_name", "license_number"))
+
+
+class DriverLicenseUpdateForm(forms.ModelForm, DriverValidationLicenseNumber):
```

Comment:
Similar to the `DriverCreationForm`, the `DriverValidationLicenseNumber` should not be used as a mixin here. Move the validation logic into the `clean` method of the form.